mod/admin.php

```diff
@@ -746,11 +746,21 @@ function admin_page_federation(App $a)
 function admin_page_queue(App $a)
 {
 	// get content from the queue table
+	// PLEASE REVIEW (not 100% sure about my code)
```

Comment:
This function does not work that way. Best is - I guess - if you create some small test script and you just play with it.

Reply:
I'll make this DBA::Select() a comment, re-enable the original query, and handle conversion within a foreach :)